further_link/__main__.py

```diff
@@ -25,8 +41,70 @@ def port():
     return int(os.environ.get("FURTHER_LINK_PORT", 8028))
 
 
-def create_app():
+def _status():
+    return "OK"
+
+
+def _version():
+    return dumps({"version": __version__})
+
+
+async def status(_):
+    return web.Response(text=_status())
+
+
+async def version(_):
+    return web.Response(text=_version())
+
+
+async def create_bluetooth_app() -> Optional[BluetoothDevice]:
+    bluetooth_device = None
+    try:
+        # Associate characteristic read/write with handlers
+        # 'upload' and 'run' features use 2 characteristics; one ('READ') is used by the client to read the
+        # characteristic's value, or get notified of a change in the characteristic's value when subscribed.
+        #
+        # The other ('WRITE') is used by the client to send a value to the server.
+        #
+        # When a value is written to the 'WRITE' characteristic, the associated callback will be executed,
+        # writing a value to the associated 'READ' characteristic.
+        #
+        # In both cases, the value written/read from/to the characteristic is treated as a message being sent/received.
+        gatt_config = GattConfig("Further", FURTHER_GATT_CONFIG)
+        gatt_config.register_read_handler(
+            PT_SERVICE_UUID, PT_STATUS_CHARACTERISTIC_UUID, _status
+        )
+        gatt_config.register_read_handler(
+            PT_SERVICE_UUID, PT_VERSION_CHARACTERISTIC_UUID, _version
+        )
+        gatt_config.register_write_handler(
+            PT_SERVICE_UUID, PT_APT_VERSION_CHARACTERISTIC_UUID, apt_version_bt
+        )
```

Comment:
it might be more consistent to use two characteristics for this, so there is the same pattern for all read/write response things